pkg/tpm/tspi.go

```diff
@@ -44,44 +48,51 @@ var (
 
 // TPM global struct containing runtime information
 type TPM struct {
-	device        io.ReadWriteCloser
-	specification string
-	owned         bool
-	active        bool
-	enabled       bool
+	device          io.ReadWriteCloser
+	specification   string
+	owned           bool
+	active          bool
+	enabled         bool
+	tempDeactivated bool
 }
 
 func init() {
 	err := NewTPM()
 	if err != nil {
-		//Die
+		fmt.Printf("Registering a new TPM instance threw an error: %s", err.Error())
```

Comment:
this should probably `log.Panic(...)`. But is it necessary to register the device at module initialization time?